docs/data-sources/monitor.md

```diff
@@ -1,49 +1,75 @@
 ---
-page_title: "datadog_monitor"
+page_title: "datadog_monitor Data Source - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Use this data source to retrieve information about an existing monitor for use in other resources.
 ---
 
-# datadog_monitor Data Source
+# Data Source `datadog_monitor`
 
 Use this data source to retrieve information about an existing monitor for use in other resources.
 
 ## Example Usage
 
-```
+```terraform
 data "datadog_monitor" "test" {
   name_filter = "My awesome monitor"
   monitor_tags_filter = ["foo:bar"]
 }
 ```
 
-## Argument Reference
-
--   `name_filter`: (Optional) A monitor name to limit the search.
--   `tags_filter`: (Optional) A list of tags to limit the search. This filters on the monitor scope.
--   `monitor_tags_filter`: (Optional) A list of monitor tags to limit the search. This filters on the tags set on the monitor itself.
-
-~> **NOTE:** If more or less than a single match is returned by the search, Terraform will fail. Ensure that your search is specific enough to return a single monitor.
```

Comment:
One downside to the autogeneration is that we lose these kinds of notes. If we wanted to, I can add a template that copies the base template but had this note hard-coded in. Though I don't know if its strictly necessary here.

Reply:
We can always add it in the description of the filter arguments